### DISCUSSION TITLE
How frequently does Riven check if there are newer episodes?

### DISCUSSION THREAD

**Discussion by titooo7** [rivenmedia/riven#934] [Q&A] [answered]
I added a brand new tv shows and the first episode got added. Today the episode is released so when can I expect Riven to try and get it?

I now I can force it manually if I reset the tv shows but it'd be good if it checks when the episodes is available upon release date (like sonarr does)

thank you

**Comment by dreulavelle** [comment, marked as answer]
It was set to 24 hours, but this is going to get changed.

Also, we don't check the discussions, so the best place to reach any of the dev's or any sort of support would be through the discord.

https://discord.gg/THFcfFX5BJ

**Reply by dreulavelle** [reply]
This got changed to 4 hours.
